2015-04-25 12:09 DEBUG    manager                       Figuring out config load paths
2015-04-25 12:09 DEBUG    manager                       Found config: /Users/matt/.flexget/config.yml
2015-04-25 12:09 DEBUG    manager                       Config file /Users/matt/.flexget/config.yml selected
2015-04-25 12:09 INFO     manager                       Test mode, creating a copy from database ...
2015-04-25 12:09 INFO     manager                       Test database created
2015-04-25 12:09 VERBOSE  task_queue                    There are 3 tasks to execute. Shutdown will commence when they have completed.
2015-04-25 12:09 INFO     transmission  showrss         Trying to connect to transmission...
2015-04-25 12:09 CRITICAL task          showrss         BUG: Unhandled error in plugin transmission: timed out
2015-04-25 12:09 CRITICAL manager       showrss         An unexpected crash has occurred. Writing crash report to /Users/matt/.flexget/crash_report.2015.04.25.120952457235.log
2015-04-25 12:09 WARNING  task          showrss         Aborting task (plugin: transmission)
2015-04-25 12:09 INFO     transmission  kickass         Trying to connect to transmission...
2015-04-25 12:10 CRITICAL task          kickass         BUG: Unhandled error in plugin transmission: timed out
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/flexget/task.py", line 435, in __run_plugin
    return method(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/flexget/event.py", line 22, in __call__
    return self.func(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/flexget/plugins/plugin_transmission.py", line 33, in new_f
    f(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/flexget/plugins/plugin_transmission.py", line 149, in on_task_start
    self.client = self.create_rpc_client(config)
  File "/usr/local/lib/python2.7/site-packages/flexget/plugins/plugin_transmission.py", line 81, in create_rpc_client
    cli = transmissionrpc.Client(config['host'], config['port'], user, password)
  File "/usr/local/lib/python2.7/site-packages/transmissionrpc/client.py", line 169, in __init__
    self.get_session()
  File "/usr/local/lib/python2.7/site-packages/transmissionrpc/client.py", line 828, in get_session
    self._request('session-get', timeout=timeout)
  File "/usr/local/lib/python2.7/site-packages/transmissionrpc/client.py", line 247, in _request
    http_data = self._http_query(query, timeout)
  File "/usr/local/lib/python2.7/site-packages/transmissionrpc/client.py", line 205, in _http_query
    result = self.http_handler.request(self.url, query, headers, timeout)
  File "/usr/local/lib/python2.7/site-packages/transmissionrpc/httphandler.py", line 65, in request
    response = self.http_opener.open(request, timeout=timeout)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 431, in open
    response = self._open(req, data)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 449, in _open
    '_open', req)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 1227, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 1200, in do_open
    r = h.getresponse(buffering=True)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 1073, in getresponse
    response.begin()
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 415, in begin
    version, status, reason = self._read_status()
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 371, in _read_status
    line = self.fp.readline(_MAXLINE + 1)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/socket.py", line 476, in readline
    data = self._sock.recv(self._rbufsize)
timeout: timed out
